Samples/Notifications/Push/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -0,0 +1,203 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <iostream>
+
+#include <wil/result.h>
+#include <wil/cppwinrt.h>
+
+#include <winrt/Windows.ApplicationModel.Background.h>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Microsoft.Windows.AppLifecycle.h>
+#include <winrt/Microsoft.Windows.PushNotifications.h>
+#include <winrt/Windows.Globalization.DateTimeFormatting.h>
+
+#include <MddBootstrap.h>
+
+using namespace winrt::Microsoft::Windows::AppLifecycle;
+using namespace winrt::Microsoft::Windows::PushNotifications;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Globalization::DateTimeFormatting;
+
+// To obtain an AAD RemoteIdentifier for your app,
+// follow the instructions on https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/push/push-quickstart#configure-your-apps-identity-in-azure-active-directory
+winrt::guid remoteId{ "00000000-0000-0000-0000-000000000000"}; // Replace this with your own RemoteId
+
+HRESULT LoadWindowsAppSDK()
+{
+    // Major.Minor version, MinVersion=0 to find any framework package for this major.minor version
+    const UINT32 majorMinorVersion{ 0x00010001 }; //  { Major << 16) | Minor };
+    PCWSTR versionTag{ L"" };
+    const PACKAGE_VERSION minVersion{};
+    HRESULT hr{ MddBootstrapInitialize(majorMinorVersion, versionTag, minVersion) };
```

Comment:
Does this invoke the MddBootstrap auto initializer?

Reply:
Winrt/ WinUI apps don't need to call MddBootstrap but I don't think this applies to non-winrt apps like this one, which is as barebone win32 as it can be.

Reply:
Unless the app is self-contained, it has to be bootstrapped to an installed WindowsAppSDK framework to have access to the APIs.

After reading the doc @erik0686 linked, if the app wants the bootstrap for free, all it has to do is add that property :). An app can opt out of default support and call the bootstrap APIs if they want (but who would want that).

Very happy to see this improvement to help developers :)

Reply:
done